tutorials/io/background_loading.rst

```diff
@@ -155,7 +155,7 @@ precise control over the timings.
             # poll your loader
             var err = loader.poll()
 
-            if err == ERR_FILE_EOF: # load finished
+            if err == ERR_FILE_EOF: # finished loading
```

Comment:
You probably want to capitalize the first word here, though probably someone should just make an automated script which capitalizes comments and submit a PR.

Reply:
> You probably want to capitalize the first word here, though probably someone should just make an automated script which capitalizes comments and submit a PR.

@bojidar-bg : I try to minimise/minimize the changes I make.

Re:``approve`` function, 
I think it's nice to "see" someone "in good spirits" with a sense of humour/humor. :smile: